indexer/src/main/java/au/org/aodn/esindexer/service/IndexerServiceImpl.java

```diff
@@ -196,17 +196,13 @@ protected StacCollectionModel getMappedMetadataValues(String metadataValues) thr
         }
 
         // organisation vocabs
-        List<String> processedOrganisationVocabs = vocabService.extractVocabLabelsFromThemes(stacCollectionModel.getThemes(), AppConstants.AODN_ORGANISATION_VOCABS);
-        if (!processedOrganisationVocabs.isEmpty()) {
-            stacCollectionModel.getSummaries().setOrganisationVocabs(processedOrganisationVocabs);
-        }
+        // TODO: the logics for mapping record's organisation vocabs are heavily customised for a manual approach, AI now or later? need dedicated service's method
```

Comment:
will not using `extractVocabLabelsFromThemes` method for organisation vocabs, the record's vocabs are found in different fields of the records, not "themes"